onward/app/controllers/PopularInTagDCR.scala

```diff
@@ -0,0 +1,41 @@
+package controllers
+
+import common._
+import containers.Containers
+import contentapi.ContentApiClient
+import feed.MostReadAgent
+import model._
+import play.api.libs.ws.WSClient
+import play.api.mvc._
+import renderers.DotcomRenderingService
+import services._
+
+import scala.concurrent.Future
+
+class PopularInTagDCR(
```

Comment:
Calling things `DCR` feels a bit misleading as this should be the defacto now, but I really like not branching on `ifDcr` in the existing controllers.

Maybe we can refactor this to `PopularInTag`, and the old one to `PopularInTagDeprecated`?